tests/unit/benefits-description.spec.js

```diff
@@ -23,6 +23,13 @@ describe("benefit descriptions", function () {
         })
       }
 
+      if (institution.type == "epci") {
+        it("should have a code_siren", function () {
+          expect(typeof institution.code_siren).toBe("string")
+          expect(institution.code_siren.length).toBeGreaterThan(1)
```

Comment:
On peut aussi utiliser une regex pour valider le format du code SIREN :
```suggestion
          expect(institution.code_siren).toMatch(/^(1|2){1}[0-9]{8}$/)
```

D'après le [site de l'insee](https://www.insee.fr/fr/metadonnees/definition/c2047) on sait qu'un code siren est toujours commencé de 9 chiffres et commence par un `1` ou un `2`. Hésite pas si le code te parait pas clair :)

Reply:
C'est parti :) Merci !